src/edu/stanford/rsl/tutorial/weightedtv/Perform2DwTV.java

```diff
@@ -0,0 +1,293 @@
+package edu.stanford.rsl.tutorial.weightedtv;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import ij.IJ;
+import ij.ImageJ;
+import ij.ImagePlus;
+import edu.stanford.rsl.tutorial.filters.RamLakKernel;
+import edu.stanford.rsl.conrad.data.numeric.Grid1D;
+import edu.stanford.rsl.conrad.data.numeric.Grid2D;
+import edu.stanford.rsl.conrad.data.numeric.NumericPointwiseOperators;
+import edu.stanford.rsl.conrad.geometry.Rotations;
+import edu.stanford.rsl.conrad.geometry.transforms.ScaleRotate;
+import edu.stanford.rsl.conrad.numerics.SimpleMatrix;
+import edu.stanford.rsl.conrad.utils.ImageUtil;
+import edu.stanford.rsl.tutorial.phantoms.Phantom;
+import edu.stanford.rsl.tutorial.phantoms.SheppLogan;
+import edu.stanford.rsl.tutorial.fan.FanBeamProjector2D;
+import edu.stanford.rsl.tutorial.fan.FanBeamBackprojector2D;
+
+//import TVreconstruction.FanBeamProjector2D;
+//import edu.stanford.rsl.tutorial.fan.FanBeamProjector2D;
+
+/**
+ *[1]Candes. Enhancing Sparsity by Reweighted l1 Minimization
+ * 
+ * 
+ * @author Yixing Huang
+ *
+ */
+public class Perform2DwTV {
+	static boolean isOpencl=true;
+	
+	
+	 private static String path="D:\\wTV\\";
```

Comment:
OK. Then I will delete the output log file and only print them on the screen. That's easier for me. And I will expand the names from acronyms...In the TVGradient.java and TVGradient3D.java, some methods are not used for these examples? Should I remove them?